#! /bin/sh /usr/share/dpatch/dpatch-run

## 18_vdr-1.3.31-remuxbraces.dpatch by Klaus Schmidinger <Klaus.Schmidinger@cadsoft.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes some gcc warning messages

@DPATCH@
diff -urNad vdr-1.3.31/remux.c /tmp/dpep.7mmgBA/vdr-1.3.31/remux.c
--- vdr-1.3.31/remux.c	2005-08-28 13:46:44.000000000 +0200
+++ /tmp/dpep.7mmgBA/vdr-1.3.31/remux.c	2005-08-31 21:24:47.644378176 +0200
@@ -522,18 +522,22 @@
   };
 
 int cAudioRepacker::bitRates[2][3][16] = { // all values are specified as kbits/s
+  {
   // MPEG 1, Layer I
-  0,  32,  64,  96, 128, 160, 192, 224, 256, 288, 320, 352, 384, 416, 448, -1,
+  { 0,  32,  64,  96, 128, 160, 192, 224, 256, 288, 320, 352, 384, 416, 448, -1 },
   // MPEG 1, Layer II
-  0,  32,  48,  56,  64,  80,  96, 112, 128, 160, 192, 224, 256, 320, 384, -1,
+  { 0,  32,  48,  56,  64,  80,  96, 112, 128, 160, 192, 224, 256, 320, 384, -1 },
   // MPEG 1, Layer III
-  0,  32,  40,  48,  56,  64,  80,  96, 112, 128, 160, 192, 224, 256, 320, -1,
+  { 0,  32,  40,  48,  56,  64,  80,  96, 112, 128, 160, 192, 224, 256, 320, -1 }
+  },
+  {
   // MPEG 2, Layer I
-  0,  32,  48,  56,  64,  80,  96, 112, 128, 144, 160, 176, 192, 224, 256, -1,
+  { 0,  32,  48,  56,  64,  80,  96, 112, 128, 144, 160, 176, 192, 224, 256, -1 },
   // MPEG 2, Layer II/III
-  0,   8,  16,  24,  32,  40,  48,  56,  64,  80,  96, 112, 128, 144, 160, -1,
+  { 0,   8,  16,  24,  32,  40,  48,  56,  64,  80,  96, 112, 128, 144, 160, -1 },
   // MPEG 2, Layer II/III
-  0,   8,  16,  24,  32,  40,  48,  56,  64,  80,  96, 112, 128, 144, 160, -1
+  { 0,   8,  16,  24,  32,  40,  48,  56,  64,  80,  96, 112, 128, 144, 160, -1 }
+  }
   };
 
 cAudioRepacker::cAudioRepacker(void)
@@ -590,16 +594,16 @@
      else {
         static int samplingFrequencies[2][4] = { // all values are specified in Hz
           // MPEG 1
-          44100, 48000, 32000, -1,
+          { 44100, 48000, 32000, -1 },
           // MPEG 2
-          22050, 24000, 16000, -1
+          { 22050, 24000, 16000, -1 }
           };
 
         static int slots_per_frame[2][3] = {
           // MPEG 1, Layer I, II, III
-          12, 144, 144,
+          { 12, 144, 144 },
           // MPEG 2, Layer I, II, III
-          12, 144,  72
+          { 12, 144,  72 }
           };
 
         int mpegIndex = 1 - id;
